### ISSUE TITLE
OSD shouldn't automatically show when activating window

### ISSUE THREAD

**Issue by tofagerl** [plexinc/plex-media-player#236]
This is another issue with multitasking. When I want to ffwd or rewind a bit, I first have to activate the PMP, then wait until the OSD menu goes away, THEN I can use the arrow buttons. 
I know I could just use the arrow keys on the OSD to pick the forward and backward buttons, but they're not as good as the keyboard shortcuts. 

Two simple solutions to this that I can think of: Either make a specific keybind to open the OSD (enter?), or make a specific keybind to close the OSD (escape?). Both will probably lead to issues with how people navigate it, but I don't know how other people use PMP, so you will have to figure that part out on your own.

**Comment by tru** [comment]
"Activate" do you mean by just focusing PMP the osd shows?

**Comment by tofagerl** [comment]
No, I have to click the actual video part of the window. The titlebar is "safe".

**Comment by LongChair** [comment]
@tofagerl : is this still relevant in last builds ?

**Comment by LongChair** [comment]
@markus101 : any thoughts about this one ?
